src/components/CurrencySelector.tsx

```diff
@@ -44,21 +45,25 @@ function CurrencySelector(props: Props) {
   return (
     <div className="field">
       <Label htmlFor={inputRef.current?.id} required aria-required>
-        Select your bank account currency:
+        {props.label}
       </Label>
       <Combobox
         value={props.value}
         aria-disabled={props.disabled}
         disabled={props.disabled}
         onChange={props.onChange}
         as="div"
-        className={`relative items-center grid grid-cols-[1fr_auto] field-container ${props.classes.combobox}`}
+        className={`relative items-center grid grid-cols-[1fr_auto] field-container ${
+          props.classes?.combobox || ""
+        }`}
       >
         <Combobox.Input
           ref={inputRef}
           className="w-full border-r border-gray-l3 dark:border-bluegray px-4 py-3.5 text-sm leading-5 text-gray-900 focus:ring-0"
           displayValue={(currency: WiseCurrency) =>
-            `${currency.code} - ${currency.name}`
+            !!currency.name
```

Comment:
displayValue should be typed as `Currency` instead of `WiseCurrency`

Reply:
good catch